mapstore/geostore-datasource-ovr.properties

```diff
@@ -0,0 +1,24 @@
+#Default Postgres Connection values 
+geostoreDataSource.driverClassName=org.postgresql.Driver
+geostoreDataSource.url=jdbc:postgresql://database:5432/georchestra
+geostoreDataSource.username=georchestra
+geostoreDataSource.password=georchestra
+geostoreVendorAdapter.databasePlatform=org.hibernate.dialect.PostgreSQLDialect
+geostoreEntityManagerFactory.jpaPropertyMap[hibernate.hbm2ddl.auto]=validate
+geostoreEntityManagerFactory.jpaPropertyMap[hibernate.default_schema]=geostore
```

Comment:
Schema should have the same name as the app => `mapstore` rather than `geostore`

Reply:
[this](https://github.com/georchestra/datadir/pull/186/commits/9ba3bac9bea7dfd6d3c899ee93aeaf6a049f058a) commit fixes it